INSTRUCTIONS.md

```diff
@@ -0,0 +1,26 @@
+# Instructions
+
+Most of the content on this guide is following the (VSCode extension documentation)[https://code.visualstudio.com/api/get-started/your-first-extension]. You should at least, read the `GET STARTED` session before move on.
+
+## Development
+
+To start the development you have to clone this repo and install the dependencies using `yarn`. 
+```bash
+yarn install
+```
+
+To use the extension during development you should go to `Debug and Run > Play`. It will open a new VSCode window with the extension installed.
```

Comment:
```suggestion
To use the extension during development, go to `Debug and Run > Play`. This will open a new VS Code window with the extension installed.
```